pumpkin/src/block/blocks/candle_cakes.rs

```diff
@@ -18,26 +20,27 @@ use crate::{
     entity::player::Player,
     world::World,
 };
-
-const CANDLE_MAP: [(&Item, &Block); 17] = [
-    (&Item::CANDLE, &Block::CANDLE_CAKE),
-    (&Item::WHITE_CANDLE, &Block::WHITE_CANDLE_CAKE),
-    (&Item::ORANGE_CANDLE, &Block::ORANGE_CANDLE_CAKE),
-    (&Item::MAGENTA_CANDLE, &Block::MAGENTA_CANDLE_CAKE),
-    (&Item::LIGHT_BLUE_CANDLE, &Block::LIGHT_BLUE_CANDLE_CAKE),
-    (&Item::YELLOW_CANDLE, &Block::YELLOW_CANDLE_CAKE),
-    (&Item::LIME_CANDLE, &Block::LIME_CANDLE_CAKE),
-    (&Item::PINK_CANDLE, &Block::PINK_CANDLE_CAKE),
-    (&Item::GRAY_CANDLE, &Block::GRAY_CANDLE_CAKE),
-    (&Item::LIGHT_GRAY_CANDLE, &Block::LIGHT_GRAY_CANDLE_CAKE),
-    (&Item::CYAN_CANDLE, &Block::CYAN_CANDLE_CAKE),
-    (&Item::PURPLE_CANDLE, &Block::PURPLE_CANDLE_CAKE),
-    (&Item::BLUE_CANDLE, &Block::BLUE_CANDLE_CAKE),
-    (&Item::BROWN_CANDLE, &Block::BROWN_CANDLE_CAKE),
-    (&Item::GREEN_CANDLE, &Block::GREEN_CANDLE_CAKE),
-    (&Item::RED_CANDLE, &Block::RED_CANDLE_CAKE),
-    (&Item::BLACK_CANDLE, &Block::BLACK_CANDLE_CAKE),
-];
+lazy_static! {
```

Comment:
what is the point of this? the old version still compiles and doesn't need a dependency.